storybook-example/stories/SignInWithEsignet.stories.ts

```diff
@@ -2,6 +2,35 @@ import { Meta, StoryObj } from "@storybook/web-components";
 import { SignInWithEsignet } from "./";
 import { html } from "lit";
 
+const oidcConfig = {
+  authorizeUri: "https://esignet.dev.mosip.net/authorize",
+  redirect_uri: "https://healthservices.dev.mosip.net/userprofile",
+  client_id: "88Vjt34c5Twz1oJ",
+  scope: "openid profile",
+  nonce: "ere973eieljznge2311",
+  state: "eree2311",
+  acr_values:
+    "mosip:idp:acr:generated-code mosip:idp:acr:biometrics mosip:idp:acr:static-code",
+  claims_locales: "en",
+  display: "page",
+  prompt: "consent",
+  max_age: 21,
+  ui_locales: "en",
+};
+
+const buttonConfig = {
+  type: "standard",
+  theme: "outline",
+  shape: "sharp_edges",
+  labelText: "Sign in with e-Signet",
+};
+
+const oidcConfigType =
+  '{\r\n  authorizeUri: "string",\r\n  redirect_uri: "string",\r\n  client_id: "string",\r\n  scope: "string",\r\n  nonce: "string",\r\n  state: "string",\r\n  acr_values: "string",\r\n  claims_locales: "string",\r\n  ui_locales: "string",\r\n  display: "page" | "popup" | "touch" | "wap",\r\n  max_age: "number",\r\n  prompt: "none" | "login" | "consent" | "select_account",\r\n}';
```

Comment:
why do you need /r/n?

Reply:
Added /r/n for the breaking the line and starting the new line

Reply:
If we add /r/n then it looks like proper json format
<img width="726" alt="Screenshot 2023-07-13 113653" src="https://github.com/mosip/esignet-plugins/assets/50859121/ae6e566a-c81d-4ab4-bf01-f8c3229d3c67">